proposals/Level-1.md

```diff
@@ -299,14 +320,17 @@ encoded above.
 
 ##### except_type
 
-An exception is described by its exception type signature, which corresponds to
-the data fields of the exception.
+Each exception definition (defining an exception tag) has a type signature,
+which corresponds to the data fields of the exception.
 
 | Field | Type | Description |
 |-------|------|-------------|
 | `count` | `varuint32` | The number of types in the signature |
 | `type` | `value_type*` | The type of each element in the signature |
 
+Note: An `except_type` is not actually a type, just an exception definition that
```

Comment:
Well, the exception type in an exn def is indeed the type of the defined tag, so I believe this note isn't necessary (and probably misleading).

Reply:
Removed.